include/mculib/adf4350.hpp

```diff
@@ -17,37 +17,42 @@ namespace ADF4350 {
 		int denominator = 32;
 		int O = 1;
 		int R = 1;
-		int rfPower = 0b01;
-		int auxPower = 0b00;
+		bool pdwn = false;
+		uint8_t rfPower = 0b01;
+		static constexpr int auxPower = 0b00;
```

Comment:
These settings need to remain changeable, is there a reason you want it constant?

Reply:
I just made everything constant that was possible just. I did so to clarify code and reduce RAM usage.
But please feel free to drop the const where needed.